pkg/cmd/subcommand/install.go

```diff
@@ -127,14 +129,18 @@ The permitted parameters are: --async, --knative, --shipwright, --cert-manager,
 func (i *Install) ValidateArgs(cmd *cobra.Command, args []string) error {
 	ti := util.NewTaskInformer("")
 
-	if !util.IsInSlice(i.OpenFunctionVersion, availableVersions) {
-		errMsg := fmt.Sprintf(
-			"version %s is not in the list of available versions: %v",
-			i.OpenFunctionVersion,
-			strings.Join(availableVersions[:], ", "),
-		)
-		return errors.New(ti.TaskFail(errMsg))
+	if valid, err := common.IsVersionValid(i.OpenFunctionVersion); err != nil {
+		return errors.New(ti.TaskFail(err.Error()))
+	} else {
+		if !valid {
+			return errors.New(ti.TaskFail(fmt.Sprintf(
+				"the given version %s is less than the base version %s",
```

Comment:
the given version %s is less than the base version %s
=>
the specified version %s is lower than the supported version %s